### PR TITLE
Updating pipeline docs for workshop

### PR DESCRIPTION
**Pull request by suppathak**
## Related Issues and Dependencies
 #540

### CODE REVIEW

File: docs/workshop/ml_pipeline.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 # Create an ML Pipeline using Elyra and Kubeflow Pipelines
```
Comment (oindrillac) [inline docs/workshop/ml_pipeline.md:2]: In this section, we will learn how to run the training pipeline of the ML model in automation.

To learn more about the training process refer to the chapter on [model development](./model_development.md).
```suggestion

```

File: docs/workshop/ml_pipeline.md [hunk 2]
```diff
@@ -1,6 +1,6 @@
...
 
-In order to run the above notebooks in automation, we can configure a Kubeflow Pipeline using the Elyra notebook pipelines editor from Jupyterlab
+In order to run the above notebooks in automation, we can configure a Kubeflow Pipeline using the Elyra notebook pipelines editor from Jupyterlab.
```
Comment (oindrillac) [inline docs/workshop/ml_pipeline.md:3]:
```suggestion
In order to run the above notebooks in automation, we use Kubeflow Pipelines and the [Elyra notebook pipelines editor](https://elyra.readthedocs.io/en/stable/) from Jupyterlab.
```
```suggestion
In order to run the above notebooks in automation, we can configure a Kubeflow Pipeline using the Elyra notebook pipelines editor from Jupyterlab.https://www.kubeflow.org/docs/components/pipelines/introduction/
```

File: docs/workshop/ml_pipeline.md [hunk 3]
```diff
@@ -1,6 +1,6 @@
...
 
 ## Create kubeflow runtime configuration
```
Comment (oindrillac) [inline docs/workshop/ml_pipeline.md:6]: @suppathak can correct the link to the pipeline Open pipeline(end-to-end.pipeline) ?

**Review by oindrillac — CHANGES_REQUESTED** [review]

File: docs/workshop/ml_pipeline.md (hunk 1 above)
Comment (oindrillac) [reply docs/workshop/ml_pipeline.md:2]: can you include the above to the introduction. We are going into creating the pipeline very abruptly

**Comment by oindrillac** [comment]
We say

To run the pipeline, click on the “Run” icon on the top toolbar and give the pipeline any **name** (recommended that you use your github username so that it's easier to debug later) and select the kubeflow runtime configuration that you created earlier and hit Submit.

but the image doesnt use github username says end-to-end can be misleading, can you please replace that image

**Review by oindrillac — APPROVED** [review]
/lgtm
/approve

**Review by aakankshaduggal — COMMENTED** [review]
/lgtm
/approve

**Pull request merged**